### PR TITLE
Jeff's initial 1.7 changes

### LINKED ISSUE #59: first changes for version 1.7
These Trac tickets have been implemented for the DocBook version of the CF Conventions version 1.7.  They need to be re-implemented in the AsciiDoc version.
 ticket 61
 ticket 62 
 ticket 64
 ticket 65
 ticket 69 
For these tickets, there is no need to make corresponding changes in the conformance document.

### PR DESCRIPTION
**Pull request by rhattersley**
Just in case it helps "unstick" the 1.7 preparation process I've created this PR with the commits that @painter1 had accrued on the `1.7.1` branch on the main repo.

(Who can merge PRs? The list of [people in the cf-convention organisation](https://github.com/orgs/cf-convention/people) seems to leave out quite a few! It might help for the members to make themselves public?)

Closes #59

### CODE REVIEW

**Comment by rhattersley** [comment]
cc @JonathanGregory, @JohnLCaron, @rsignell-usgs in case any of you have merge rights.

**Comment by rhattersley** [comment]
Please could someone add the 1.7 milestone to this PR.

File: appe.adoc [hunk 1]
```diff
@@ -32,8 +32,14 @@ respect to the specified dimension.
 
 | `mean` | __u__ | Mean (average value)
 
+| `mean_of_upper_decile` | __u__ | Mean of the upper group of data values defined by the upper tenth of their distribution 
+
 | `mode` | __u__ | Mode (most common value)
 
+| `range` | __u__ | Absolute difference between maximum and minimum
+
+| `root_mean_square` | __u__ | Root mean square (RMS)
```
Comment (marqh) [inline appe.adoc]: http://cf-trac.llnl.gov/trac/ticket/61

File: appe.adoc [hunk 2]
```diff
@@ -32,8 +32,14 @@ respect to the specified dimension.
 
 | `mean` | __u__ | Mean (average value)
 
+| `mean_of_upper_decile` | __u__ | Mean of the upper group of data values defined by the upper tenth of their distribution
```
Comment (marqh) [inline appe.adoc]: http://cf-trac.llnl.gov/trac/ticket/61

File: ch07.adoc [hunk 3]
```diff
@@ -153,9 +153,9 @@ significant in the method name. Some methods (e.g., `variance` ) imply a
...
+It must be remembered that the method applies only to the axis designated in **`cell_methods`** by __name__, and different methods may apply to other axes. If, for instance, a precipitation value in a longitude-latitude cell is given the method **`maximum`** for these axes, it means that it is the maximum within these spatial cells, and does not imply that it is also the maximum in time. Furthermore, it should be noted that if any __method__ other than "**`point`**" is specified for a given axis, then **`bounds`** should also be provided for that axis (except for the relatively rare exceptions described in <<cell-methods-no-coordinates>>).
```
Comment (marqh) [inline ch07.adoc]: http://cf-trac.llnl.gov/trac/ticket/64

File: ch07.adoc [hunk 4]
```diff
@@ -153,9 +153,9 @@ significant in the method name. Some methods (e.g., `variance` ) imply a
...
...ensive (which does not). Suppose, for example, the quantities "accumulated precipitation" and "precipitation rate" each have a time axis. A variable representing accumulated precipitation is extensive in time because it depends on the length of the time interval over which it is accumulated. For correct interpretation, it therefore requires a time interval to be completely specified via a boundary variable (i.e., via a **`bounds`** attribute for the time axis). In this case the default interpretation is that the cell method is a sum over the specified time interval. This can be (optionally) indicated explicitly by setting the cell method to **`sum`**. A precipitation rate on the other hand is intensive in time and could equally well represent either an instantaneous value or a mean value over the time interval specified by the cell. In this case the default interpretation for the quantity would be "instantaneous" (which, optionally, can be indicated explicitly by setting the cell method to **`point`**). More often, however, cell values for intensive quantities are means, and this should be indicated explicitly by setting the cell method to **`mean`** and specifying the cell bounds.
```
Comment (marqh) [inline ch07.adoc]: http://cf-trac.llnl.gov/trac/ticket/64

File: ch07.adoc [hunk 5]
```diff
@@ -235,7 +235,7 @@ If the cell method applies to a combination of axes, they may have a common orig
...
 
-A dimension of size one may be the result of "collapsing" an axis by some statistical operation, for instance by calculating a variance from time series data. We strongly recommend that dimensions of size one be retained (or scalar coordinate variables be defined) to enable documentation of the method (through the **`cell_methods`** attribute) and its domain (through the **`cell_bounds`** attribute).
+A dimension of size one may be the result of "collapsing" an axis by some statistical operation, for instance by calculating a variance from time series data. We strongly recommend that dimensions of size one be retained (or scalar coordinate variables be defined) to enable documentation of the method (through the **`cell_methods`** attribute) and its domain (through the **`bounds`** attribute).
```
Comment (marqh) [inline ch07.adoc]: http://cf-trac.llnl.gov/trac/ticket/64

File: appe.adoc [hunk 6]
```diff
@@ -32,8 +32,14 @@ respect to the specified dimension.
 
 | `mean` | __u__ | Mean (average value)
 
+| `mean_of_upper_decile` | __u__ | Mean of the upper group of data values defined by the upper tenth of their distribution 
+
 | `mode` | __u__ | Mode (most common value)
 
+| `range` | __u__ | Absolute difference between maximum and minimum
```
Comment (marqh) [inline appe.adoc]: http://cf-trac.llnl.gov/trac/ticket/65

File: appd.adoc [hunk 7]
```diff
@@ -252,7 +252,56 @@ formula_terms = "s: var1 eta: var2 depth: var3 a: var4 b: var5 depth_c: var6"
 
  
 
-  
+=== Ocean s-coordinate, generic form 1
```
Comment (marqh) [inline appd.adoc]: http://cf-trac.llnl.gov/trac/ticket/93

File: ch05.adoc [hunk 8]
```diff
@@ -393,6 +393,73 @@ variables:
 
 
 
+[[use-of-the-crs-well-known-text-format, Section 5.6.1, "Use of the CRS Well-known Text Format"]]
```
Comment (marqh) [inline ch05.adoc]: http://cf-trac.llnl.gov/trac/ticket/69

File: appf.adoc [hunk 9]
```diff
@@ -297,6 +297,11 @@ for numeric.
 |===============
 | Attribute | Type | Description
 
+| **`crs_wkt`** | S
```
Comment (marqh) [inline appf.adoc]: http://cf-trac.llnl.gov/trac/ticket/69

**Comment by marqh** [comment]
this covers all the changes required by the referenced trac tickets and no unreferenced changes

thank you
@painter1 @rhattersley

**Comment by rhattersley** [comment]
:tada: Our first not-trivial change. :grin:

**Pull request merged**